code/_helpers/text.dm

```diff
@@ -207,6 +207,15 @@
 		t = "[t] "
 	return t
 
+/proc/add_characters(c, n = 0) //Adds whatever character in 'c' and repeats that process 'n' times then returns text string as "XXXXXXXXXX"
```

Comment:
I don't see this proc being called anywhere. Am I missing something?

Reply:
Removed it. It was meant to be used for nuclear codes.